src/converters/toCamelCase.ts

```diff
@@ -7,7 +7,7 @@ export function toCamelCaseValue(data: unknown): unknown {
     } else if (Array.isArray(data)) {
         return data.map(toCamelCaseValue);
     } else {
-        const result = {};
+        const result: Record<string, any> = {};
```

Comment:
Does this work?
```suggestion
        const result: Record<string, unknown> = {};
```

Reply:
Yup, that works! I updated the line just now.